plugin/pkg/scheduler/algorithm/priorities/util/non_zero.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
I didn't what you mean.

Let me clarify my cycle:
- priorities import schedulercache
- if this part isn't separated out, schedulercache will import priorities.